tests/console/yast2_ftp.pm

```diff
@@ -14,8 +14,55 @@ use strict;
 use base "console_yasttest";
 use testapi;
 use utils;
+use version_utils;
+use y2logsstep;
+
+sub vsftd_setup_checker {
+    my $config_ref              = pop();
+    my @vsftpd_conf_tested_dirs = qw(pasv_min_port pasv_max_port anon_mkdir_write_enable anon_root anon_umask anon_upload_enable anon_max_rate chroot_local_user
+      ftpd_banner local_root local_umask local_max_rate max_clients max_per_ip pasv_enable rsa_cert_file ssl_sslv2 ssl_sslv3 ssl_tlsv1);
+
+    foreach (@vsftpd_conf_tested_dirs) {
+        if (script_run("grep \"^$_=$config_ref->{$_}\" \/etc\/vsftpd\.conf")) {
+            script_run("echo \"$_=$config_ref->{$_}\" >> /tmp/failed_vsftpd_directives.log");
+        }
+    }
+    assert_script_run("[[ ! -e /tmp/failed_vsftpd_directives.log ]]");
```

Comment:
I still find it confusing, as if someone new would review such failure, she or he may think that it's about missing file. What if we simply use simple string in L#27 
```perl
$error .= "vsftpd directory $_ not found in config file\n"
```
and the die if $error, similarly as you've done in vsftpd_firewall_checker method. WDYT?

Reply:
That is a good idea. What about this ?
    foreach (@vsftpd_conf_tested_dirs) {
        if (script_run("grep \"^$_=$config_ref->{$_}\" \/etc\/vsftpd\.conf")) {
            $error .= "vsftpd directive \"$_=$config_ref->{$_}\" not found in \/etc\/vsftpd\.conf"
        }
    }
    die "Missing vsftpd\.conf directives" if $error;

Reply:
looks good, just add code to init variable and push it. We don't need verification run for such a change.